app/grants/models.py

```diff
@@ -513,7 +513,7 @@ def successful_contribution(self, tx_id):
         """Create a contribution object."""
         from marketing.mails import successful_contribution
         self.last_contribution_date = timezone.now()
-        self.next_contribution_date = timezone.now() + timezone.timedelta(seconds=int(self.real_period_seconds))
+        self.next_contribution_date = timezone.now() + timezone.timedelta(seconds=float(self.real_period_seconds))
```

Comment:
Quick question...  why are we dealing in floats/ints on decimalfields? Why are we even storing these as decimals if we're not using them as decimals in most places?

Reply:
I don't have a good answer. This as likely due to my ignorance of django/python when creating these fields. What type should be used?

Reply:
Is this a blocker for this PR going out today?